types/faker/faker-tests.ts

```diff
@@ -65,18 +66,19 @@ resultDate = faker.date.soon();
 resultDate = faker.date.soon(30);
 resultStr = faker.date.month();
 resultStr = faker.date.month({
-	abbr: true,
-	context: true
+    abbr: true,
```

Comment:
Whitespace changes and trailing commas? Not needed.

Reply:
@mattbishop done.